### DISCUSSION TITLE
Is there a way to downgrade realm version?

### DISCUSSION THREAD

**Discussion by Sr3qt** [ppy/osu#24666] [Q&A] [answered]
I recently cloned the master branch off github and after running that version, my normal lazer installation said the realm file was too new and that I had to update lazer to the newest version. None of my files loaded. However, I am already using the newest lazer release 2023.815.

Is there a way to downgrade my realm files so that my normal osu! lazer installation recognizes it or in another way fix this?

**Comment by peppy** [comment, marked as answer]
There is not. Wait for the next release.

Of note, if you run debug the game will store one realm file per version. If you're running release then this is a great warning to not do so in the future.
